src/lib/parameters/px4params/jsonout.py

```diff
@@ -30,36 +30,39 @@ def __init__(self, groups, board, inject_xml_file_name):
         last_param_name = ""
         board_specific_param_set = False
         for group in groups:
-            group_name=group.GetName()
+            group_name=group.name
 
             def get_typed_value(value: str, type_name: str):
                 if type_name == 'Float': return float(value)
                 if type_name == 'Int32': return int(value)
                 return value
 
-            for param in group.GetParams():
-                if (last_param_name == param.GetName() and not board_specific_param_set) or last_param_name != param.GetName():
+            for param in sorted(group.parameters):
+                if (last_param_name == param.name and not board_specific_param_set) or last_param_name != param.name:
                     curr_param=dict()
-                    curr_param['name'] = param.GetName()
-                    type_name = param.GetType().capitalize()
+                    curr_param['name'] = param.name
+                    type_name = param.type.capitalize()
                     curr_param['type'] = type_name
                     if not type_name in allowed_types:
                         print("Error: %s type not supported: curr_param['type']" % (curr_param['name'],curr_param['type']) )
                         sys.Exit(1)
-                    curr_param['default'] = get_typed_value(param.GetDefault(), type_name)
+                    curr_param['default'] = get_typed_value(param.default, type_name)
 
                     curr_param['group'] = group_name
-                    if param.GetCategory():
-                        curr_param['category'] = param.GetCategory()
+                    if param.category != "":
+                        curr_param['category'] = param.category
                     else:
                         curr_param['category'] = 'Standard'
 
-                    if param.GetVolatile():
+                    if param.volatile != "":
```

Comment:
volatile isn't a string
(fixed in a later commit, but should be amended to this one)